handbook/company/communications.md

```diff
@@ -154,31 +143,54 @@ Team members can request reimbursement through [Gusto]([https://app.gusto.com/ex
 
+#### Travel expenses
+Travel expenses are any purchases made that support the attendance of a Fleetie at a company event, customer visit, conference attendance, or other work-required travel. These expenses are considered [non-personnel expenses](https://docs.google.com/spreadsheets/d/1X-brkmUK7_Rgp7aq42drNcUg8ZipzEiS153uKZSabWc/edit#gid=2112277278), and are not "individualized". Travel expenses are allocated to a specific event spend. In order to allocate this spend correctly, Fleeties must follow the [expense acquittal process](#expense-aquittal).
+For more guidance on travel, see [Company travel](#company-travel).
+
+#### One-off expenses
+One-off expenses are non-travel, non-recurring expenses. These expenses are considered [non-personnel expenses](https://docs.google.com/spreadsheets/d/1X-brkmUK7_Rgp7aq42drNcUg8ZipzEiS153uKZSabWc/edit#gid=2112277278). There are a varity of reasons a Fleetie might make a one-off expense, including purchase of headphones or ergonomic workstation needs, printing or sending of work documents, purchase of a swag item, etc. 
+One-off expenses should always be made with the Fleetie's Brex card, and aquitted using the [expense acquittal process](#expense-aquittal)
+
+#### Non-travel purchases that exceed a Brex cardholder's limit
+For non-travel purchases that would require an increase in the Brex cardholder's limit, please [make a request](https://fleetdm.com/handbook/business-operations#intake) with following information:
+- The nature of the purchase (i.e. SaaS subscription and what it's used for)
+- The cost of the purchase and whether it is a fixed or variable (i.e. use-based) cost.
+- Whether it is a one time purchase or a recurring purchase and at what frequency the purchase will re-occur (annually, monthly, etc.)
+- If there are more ideal options to pay for the purchase (i.e. bill.com, the Fleet AP Brex card, etc.) that method will be used instead.  
+- In general, recurring purchases such as subscription services that will continually stretch the spend limit on a cardholder's Brex card should be paid through other means. 
+- For one time purchases where payment via credit card is the most convenient then the card limit will be temporarily increased to accomodate the purchase.  
+
+#### Expense aquittal process
+Fleeties are encourage to [spend company money like it's their own](#spending-company-money), however to ensure that spend is captured correctly, teammates need to take the following steps after spending money:
+- If the spend is under $50, Fleeties can use the memo field in the Brex expense details to provide context on what the spend was for.
+- If the spend is over $50, Fleeties must attach a receipt for the item and add a memo to provide context on what the spend was for.
+- **In addition:** If the spend is for travel, include the event name and year as the prefix, e.g: "DEFCON 2023 - Uber from airport to hotel"
```

Comment:
The current policy of memos only has been working well enough.  Let's discuss before changing so that I can get a better understanding of where receipts are more helpful than memos for this kind of expense.